javascript-data-model/order-history.js

```diff
@@ -0,0 +1,54 @@
+var orderHistory = [
+  {
+    orderPlaced: 'August 4, 2020',
+    orderNumber: '#114-3941689-8772232',
+    orderTotal: '$34.00',
+    shipTo: 'JS Masher',
+    deliveryDate: 'Delivered August 8, 2020',
+    dropStatus: null,
+    itemDescription: ['JavaScript for impatient programmers'],
+    authorLastName: ['Rauschmayer'],
+    authorFirstName: ['Dr. Axel'],
+    returnWindowStatus: ['Return window closed on Sep 7, 2020'],
+    itemCost: ['$31.55']
```

Comment:
Instead of organizing the data into multiple different arrays, I would make one array with an object for each order that contains all of this data.